tests/components/airvisual/test_config_flow.py

```diff
@@ -0,0 +1,87 @@
+"""Define tests for the AirVisual config flow."""
+from unittest.mock import patch
+
+from pyairvisual.errors import InvalidKeyError
+
+from homeassistant import data_entry_flow
+from homeassistant.components.airvisual import CONF_GEOGRAPHIES, DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.const import CONF_API_KEY, CONF_LATITUDE, CONF_LONGITUDE
+
+from tests.common import MockConfigEntry, mock_coro
```

Comment:
`mock_coro` is legacy.

Reply:
https://github.com/home-assistant/core/pull/32624